ST_resgroups.c

```diff
@@ -650,7 +651,7 @@ void rgroup_Establish(void) {
 
                 /* Establishment for species that belong to annual functional groups*/
                 if (Species[sp]->max_age == 1) {
-                    //printf("Globals.currYear = %hu, call to _add_annuals sp=%d Species[sp]->lastyear_relsize : %.5f \n", Globals.currYear, sp, Species[sp]->lastyear_relsize);
+                    //printf("Globals->currYear = %hu, call to _add_annuals sp=%d Species[sp]->lastyear_relsize : %.5f \n", Globals->currYear, sp, Species[sp]->lastyear_relsize);
```

Comment:
Can we get rid of this commented-out code?

Reply:
@kpalmqui I'm fine with removing this code so the code is more readable. But if you want to keep it for testing on your end that's fine too!

Reply:
fine with me.